zebra-test/src/vectors/block.rs

```diff
@@ -383,6 +514,12 @@ lazy_static! {
     // for i in 280000 280001; do
     //     zcash-cli -testnet getblock $i 0 > block-test-$[i/1000000]-$[i/1000%1000]-00$[i%1000].txt
     // done
+    // for i in `cat post_sapling_testnet_heights`; do
+    //     zcash-cli -testnet z_gettreestate "$i" | \
+    //         jq --arg i "$i" \
+    //            --raw-output \
+    //            '"pub static ref SAPLING_FINAL_ROOT_TESTNET_\($i)_BYTES: [u8; 32] = <[u8; 32]>::from_hex(\"\(.sapling.commitments.finalRoot)\").expect(\"final root bytes are in valid hex representation\");"'
+    // done
```

Comment:
💜

Reply:
Yeah there was no way I wanted to type that all out myself